src/core/js/views/buttonsView.js

```diff
@@ -96,28 +92,29 @@ define(function() {
                 var buttonText = this.model.get('_buttons')["_" + changedAttribute].buttonText;
 
                 switch (changedAttribute) {
-                    case "showCorrectAnswer": case "hideCorrectAnswer":
+                    case "submit":
+                    case "showCorrectAnswer":
+                    case "hideCorrectAnswer":
     				    //make model answer button inaccessible but enabled for visual users
     				    //	due to inability to represent selected incorrect/correct answers to a screen reader, may need revisiting
-                        this.$('.buttons-action').a11y_cntrl(false).html(buttonText).attr('aria-label', ariaLabel);
+                        this.$('.buttons-action').a11y_cntrl_enabled(false).html(buttonText).attr('aria-label', ariaLabel);
```

Comment:
Sorry, this was my misunderstanding. I'll make the change.

I've noticed an issue around this, the buttons aren't being enabled. Do I need to call
this.$('.buttons-action').a11y_cntrl_enabled(true).a11y_cntrl(false) to ensure the button is 'enabled' but is then 'aria-hidden'?